editor.planx.uk/src/@planx/components/Content/model.ts

```diff
@@ -1,6 +1,7 @@
 import { MoreInformation, parseMoreInformation } from "../shared";
 
 export interface Content extends MoreInformation {
+  customBackground: boolean;
```

Comment:
Is there any way we could derive this information from the props already passed in as opposed to adding a new prop?

Is `customBackground` always the same as `color !== "#FFF"` or whatever the default that's set is?

Reply:
Need the pizza to build to confirm this - but _I think_ if we introduce a new required prop like this, all existing Content components will break / need to be updated which is less an than ideal! Another reason to: try to derive from existing props, or at minimum introduce this as an optional prop!

Reply:
Good shout and I appreciate the logic introduced here may not be optimal (and I hadn't considered the implications of a new prop breaking existing components).

I'll try to rework so that the padding is introduced only if the background colour is changed from the default to see if that would be a simpler solution.

Reply:
@DafyddLlyr @jessicamcinchak I've now updated this to be much simpler, doing as Daf suggested and checking whether or not the background is white rather than introducing a new option.

I am however struggling to get it working as expected, if no background colour is selected the prop is passed through empty, however trying to target this with `undefined` is not working. 

I've also tried adding `#ffffff` (equivalent white output by the colour picker) as a default prop and this is still not working.

**Note:** I've also disabled the alpha/transparency slider in the colour picker as this appears to be non-functional, at least in Chrome and Safari.

Reply:
Thanks for the update - I'll take a look @ianjon3s - checking for a negative is always hard!

Reply:
Here's a pizza with three content components:

1. Content with default background (no background colour selected)
2. Content with white background selected
3. Content with other custom background colour selected

https://2429.planx.pizza/testing/content-colour-test/preview

Reply:
Pizza looks great - thanks for the link ✅